boxcli/midcobra/telemetry.go

```diff
@@ -78,15 +80,35 @@ func (m *telemetryMiddleware) postRun(cmd *cobra.Command, args []string, runErr
 		return // Ignore invalid commands
 	}
 
+	var sentryEventID string
+	if runErr != nil {
+		defer sentry.Flush(2 * time.Second)
+		sentryEventID = string(*sentry.CaptureException(runErr))
+	}
+
 	trackEvent(segmentClient, &event{
-		AppName:     m.opts.AppName,
-		AppVersion:  m.opts.AppVersion,
-		Command:     subcmd.CommandPath(),
-		CommandArgs: subargs,
-		DeviceID:    deviceID(),
-		Duration:    time.Since(m.startTime),
-		Failed:      runErr != nil,
-		Packages:    getPackages(cmd),
+		AppName:       m.opts.AppName,
+		AppVersion:    m.opts.AppVersion,
+		Command:       subcmd.CommandPath(),
+		CommandArgs:   subargs,
+		DeviceID:      deviceID(),
+		Duration:      time.Since(m.startTime),
+		Failed:        runErr != nil,
+		Packages:      getPackages(cmd),
+		SentryEventID: sentryEventID,
+	})
+}
+
+func initSentry(opts TelemetryOpts) {
+	sentrySyncTransport := sentry.NewHTTPSyncTransport()
+	sentrySyncTransport.Timeout = time.Second * 2
+	release := opts.AppName + "@" + opts.AppVersion
+
+	_ = sentry.Init(sentry.ClientOptions{
+		Dsn:              opts.DataSourceName,
+		Release:          release,
```

Comment:
development doesn't have the sentry dsn set, so by default it skips

Reply:
Yeah, I know.

I'm just thinking:
1. The "Environment" fields is empty in Sentry right now. It would be nice if it said "production" (I think that's what we've been using elsewhere)
2. If you purposefully add a DSN and a key to test while you are improving sentry, it be nice if those said "development" (i.e. that's the case with the events currently in the sentry UI)